dataland-frontend/tests/e2e/fixtures/sfdr/SfdrDataFixtures.ts

```diff
@@ -132,6 +146,7 @@ export function generateSfdrData(nullProbability = DEFAULT_PROBABILITY): SfdrDat
         reportedIncidentsOfDiscrimination: dataGenerator.randomExtendedDataPoint(generateInt()),
         sanctionedIncidentsOfDiscrimination: dataGenerator.randomExtendedDataPoint(generateInt()),
         ceoToEmployeePayGapRatio: dataGenerator.randomExtendedDataPoint(generateFloat()),
+        excessiveCeoPayRatioInPercent: dataGenerator.randomExtendedDataPoint(generatePercentageValue()),
```

Comment:
This might have to be changed to simulate values above 100%

Reply:
This is to be covered in DALA-2783 (backend field validation)

Reply:
I will wait with creating such a subtask until the strategy we came up with is approved by Julia. You can keep this comment open until I respond again

Reply:
Julia approved our suggest approach of having a standard validation for percentage fields to be between [0,100], except a hint in the options column of the devtools specifies to overwrite that. In general, nothing will break by still using values in the interval [0,100] in the fixtures, however it would be more consistent to allow the generation of larger values at those special fields. Subtask is created in DALA-2783

Reply:
ok, then i will close this here